src/int/div.rs

```diff
@@ -0,0 +1,21 @@
+//! [`Int`] division operations.
+
+use subtle::{Choice, ConstantTimeEq, CtOption};
+
+use crate::int::Int;
+use crate::{CheckedDiv, Uint};
+
+impl<const LIMBS: usize> CheckedDiv for Int<LIMBS> {
```

Comment:
the zero has two representation, I think we should handle this as well, i.e "-0"/whatever should be "-0" not "+0".